src/commands/suggest.ts

```diff
@@ -0,0 +1,39 @@
+// Codey suggest Command
+import Discord from 'discord.js';
+import { openDB } from '../components/db';
+
+export const suggestCmd = async (message: Discord.Message, args: string[]) => {
+  const db = openDB();
+  const state = 'create';
```

Comment:
I'd prefer if we have some kind of enum for states, since we will start adding more https://www.typescriptlang.org/docs/handbook/enums.html